app/builders/labelling_form_builder.rb

```diff
@@ -0,0 +1,75 @@
+class LabellingFormBuilder < ActionView::Helpers::FormBuilder
+  include ActionView::Helpers::CaptureHelper
+  include ActionView::Helpers::TagHelper
+  include ActionView::Context
+
+  def form_group(attribute, options={}, &block)
+    a = Attribute.new(self, attribute)
+    FormGroup.new(self, a, options).content capture(&block)
+  end
```

Comment:
As all fields need to be wrapped by a form-group element this can be moved into the base class that TextField extends